archivebox/core/models.py

```diff
@@ -50,6 +60,13 @@ def as_link(self) -> Link:
     def as_link_with_details(self) -> Link:
         from ..index import load_link_details
         return load_link_details(self.as_link())
+    
+    def get_tags_str(self) -> str:
+        tags = ','.join(
+            tag.name
+            for tag in self.tags.all()
+        ) if self.tags.all() else ''
```

Comment:
```suggestion
        tags = ','.join(self.tags.values_list('name', flat=True)) if self.tags.exists() else ''
```
minor change, this is a migration so you can ignore it, but for future code this pattern is faster ^